src/dialects/mysql/query-generator-typescript.ts

```diff
@@ -12,4 +15,27 @@ export class MySqlQueryGeneratorTypeScript extends AbstractQueryGenerator {
   showIndexesQuery(tableName: TableNameOrModel) {
     return `SHOW INDEX FROM ${this.quoteTable(tableName)}`;
   }
+
+  removeIndexQuery(tableName: TableNameOrModel, indexNameOrAttributes: string | string[], options: RemoveIndexQueryOptions) {
```

Comment:
In this signature, options has been made mandatory, should remain optional (on phone so I didn't check but could be the case for the others too)